test/e2e/support.js

```diff
@@ -0,0 +1,45 @@
+import '@agoric/synpress/support/index';
+import { AGORIC_NET } from './test.utils';
+
+Cypress.Commands.add('addKeys', (params) => {
+  const { keyName, mnemonic, expectedAddress } = params;
+  const command = `
+    expect -c "
+      spawn agd keys add ${keyName} --recover --keyring-backend=test
+      expect {
```

Comment:
TIL `expect`!

though I'm surprised this command doesn't have a non-interactive mode. the help says interactive isn't the ddefault,
```
  -i, --interactive              Interactively prompt user for BIP39 passphrase and mnemonic
```

Reply:
I think running the command using non-interactive mode creates a new mnemonic phrase. My goal is to recover existing wallets because they hold enough balances to place bids.

I made an update that eliminates the use of `expect`. You can view the change [here](https://github.com/Agoric/wallet-app/compare/c2d7914a65314845bf56ddb1f53207c60c311752..1cb5aaef9ddd0c0d47ffdf8580f9aae3d366ae71)

Reply:
Yeah that's better.

If you want the command to succeed even if the key already exits you can add `|| true`